config/app/src/main/java/com/google/samples/quickstart/config/kotlin/MainActivity.kt

```diff
@@ -47,6 +50,24 @@ class MainActivity : AppCompatActivity() {
         remoteConfig.setDefaultsAsync(R.xml.remote_config_defaults)
         // [END set_default_values]
 
+        // [START add_config_update_listener]
+        remoteConfig.addOnConfigUpdateListener(object : ConfigUpdateListener {
+            override fun onUpdate(configUpdate : ConfigUpdate) {
+                Log.d(TAG, "Updated keys: " + configUpdate.updatedKeys.joinToString(", "));
```

Comment:
```suggestion
                Log.d(TAG, "Updated keys: " + configUpdate.updatedKeys);
```

Can take advantage of `Set`'s built in `toString()`